scene/resources/style_box_texture.cpp

```diff
@@ -236,6 +236,7 @@ void StyleBoxTexture::_bind_methods() {
 	BIND_ENUM_CONSTANT(AXIS_STRETCH_MODE_STRETCH);
 	BIND_ENUM_CONSTANT(AXIS_STRETCH_MODE_TILE);
 	BIND_ENUM_CONSTANT(AXIS_STRETCH_MODE_TILE_FIT);
+	BIND_ENUM_CONSTANT(AXIS_STRETCH_MODE_CENTER);
```

Comment:
It needs to also be added to the hint string of `axis_stretch_horizontal|vertical` (like it's done in NinePatchRect).

Reply:
Sorry about the trouble